common/changes/@itwin/presentation-opentelemetry/presentation-OpenTelemetry_2022-08-19-10-54.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/presentation-opentelemetry",
+      "comment": "convert diagnostics logs to a format which can be passed to OpenTelemetry exporter",
```

Comment:
The comment should be more user friendly. This is the first changelog entry, so it should state the purpose of the package and what it contains.